tools/st2ctl

```diff
@@ -51,6 +51,7 @@ function print_usage() {
     echo "usage: st2ctl {reload, clean}"
     echo "optional arguments:"
     echo "  --register-all                Register all."
+    echo "  --register-triggers           Register all triggers."
```

Comment:
Also need to update `st2common/bin/st2ctl`, I believe this file is unused now.

@manasdk @lakshmi-kannan can some verify that this file is indeed unused and we can get rid of it to avoid duplication and confusion?